src/sage/schemes/curves/zariski_vankampen.py

```diff
@@ -380,6 +381,9 @@ def voronoi_cells(V):
 
     - ``V`` -- a corrected Voronoi diagram
 
+    - ``vertical_lines`` -- list (default: ``[]``); indices of the
+      vertical lines
```

Comment:
This could be better as a (frozen)set (or stated as something equivalent) considering the containment testing used.

Reply:
Let me think about it

Reply:
I changed it to a `tuple` but I do not know if it is what you mean.

Reply:
Done.